### PR TITLE
add border

### PR DESCRIPTION
**Pull request by Fme82**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Git Demo</title>
+    <link href="https://www.skylabcoders.com/skylab-coders-academy_408.png" type="image/png" rel="Shortcut Icon"></link>
+    <link rel="stylesheet" href="style.css" ></link>
```
Comment (rivannadev) [inline index.html:9]: check code!!

File: index.html [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+</head>
+<body>
+    <h1>Git Demo</h1>
```
Comment (Fme82) [inline index.html]: gdg

Comment (rivannadev) [reply index.html]: I'll check it!!

File: index.html (hunk 1 above)
Comment (Fme82) [reply index.html:9]: Revisado

File: index.html [hunk 3]
```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html lang="en">
```
Comment (rivannadev) [inline index.html:2]: ok!

File: index.html [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Git Demo</title>
+    <link href="https://www.skylabcoders.com/skylab-coders-academy_408.png" type="image/png" rel="Shortcut Icon"></link>
+    <link rel="stylesheet" href="style.css" ></link>
+</head>
+<body>
```
Comment (Fme82) [inline index.html:11]: 👍🏻

Comment (rivannadev) [reply index.html:11]: perfect!

**Review by rivannadev — APPROVED** [review]

**Pull request merged**